researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -16,23 +18,26 @@
 import org.researchstack.backbone.utils.LogExt;
 import org.researchstack.backbone.utils.ObservableUtils;
 import org.researchstack.skin.AppPrefs;
-import org.researchstack.skin.DataProvider;
-import org.researchstack.skin.DataResponse;
-import org.researchstack.skin.ResourceManager;
-import org.researchstack.skin.model.SchedulesAndTasksModel;
+import org.researchstack.backbone.DataProvider;
```

Comment:
I think we should see how things evolve and then do a clean up or re-organization once we move back into master.

As of right now, my thoughts for moving forward are that we should move everything from Skin into backbone when we can, and eventually delete Skin all together.  Backbone should have an abstract DataProvider, which encompasses every possible Data operation.  There should be a LocalDataProvider in backbone, which is a concrete implementation of DataProvider, that stores everything locally, and also is something for the developers to use as guidance on implementing  the app, or can be used if the app requires no internet access.  BridgeSDK would still have BridgeDataProvider, which would extend, or simply use LocalDataProvider, to also do all the server operations.